.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
```

Comment:
this file is only in template for nf-core pipelines

Reply:
Done